src/commonMain/kotlin/exchange.dydx.abacus/output/Market.kt

```diff
@@ -195,30 +197,32 @@ data class MarketConfigs(
                 existing.basePositionNotional != basePositionNotional ||
                 existing.candleOptions != candleOptions ||
                 existing.perpetualMarketType != perpetualMarketType ||
-                existing.v4 != v4
+                existing.v4 != v4 ||
+                existing.effectiveInitialMarginFraction != effectiveInitialMarginFraction
             ) {
                 MarketConfigs(
-                    clobPairId,
-                    largeSize,
-                    stepSize,
-                    tickSize,
-                    stepSize.numberOfDecimals(),
-                    tickSize.numberOfDecimals(),
-                    displayStepSize,
-                    displayTickSize,
-                    displayStepSize.numberOfDecimals(),
-                    displayTickSize.numberOfDecimals(),
-                    minOrderSize,
-                    initialMarginFraction,
-                    maintenanceMarginFraction,
-                    incrementalInitialMarginFraction,
-                    incrementalPositionSize,
-                    maxPositionSize,
-                    basePositionNotional,
-                    baselinePositionSize,
-                    candleOptions,
-                    perpetualMarketType,
-                    v4,
+                    clobPairId = clobPairId,
```

Comment:
Not sure if you added those by hand.. But I just found that Android Studio can do this for you with two-clicks.

Reply:
yet another reason to download android studio!